.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I general don't  like committing IDE settings. What's the advantage o sharing this? How hard it it to generate it?

Reply:
Only IDE related commits could be those, that are inside `.gitignore` saying to ignore certain IDE related files.

Reply:
Mmm I don't like having project gitignore assuming a given IDE file.

I prefer to have disciplined contributors configuring  their global gitignores according to the tool they use.

I think project gitignore should be limited to build generated  files.

Maybe it is just a matter of taste - I don't mind being on the minority side for that topic.

Reply:
Me too, but this time the goal is to be able to run Jest tests easily. So this file can help other people in the project

Reply:
Partially true. Because if one is using something other than VSCode, then given file will just be random file in project, that doesn't benefit work.

Reply:
(BTW coming from the java world I don't get why one would need config to run unit tests...)

Reply:
Agreed with @chibenwa and @jeserkin here. Not everybody uses the same IDE, trying to impose an IDE by pushing file settings specific to one is not a good idea IMO

Reply:
I intentionally pushed this file and mentioned the configuration in the CONTRIBUTING.md.
VSCode is not needed to run tests, and there are also ways to debug with other IDEs.
I just think this file can be useful for VSCode users.
Also, the .vscode directory is hidden directory.
I can make a separate PR, what do you think about it?